### ISSUE TITLE
[FP] 'compact(...)' variables existence with variable

### ISSUE THREAD

**Issue by orklah** [kalessil/phpinspectionsea#1153]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Issue type     | False-Positive                        |
| Plugin         | Php Inspections (EA Extended)  |

Current behaviour (description/screenshot:)
--
The following code:
```php
<?php
function foo() {
    $variable_name = 'variable_value';
    $variable_name_container = 'variable_name';

    $final_array = compact($variable_name_container);

    var_dump($final_array);
}
foo();
```
Displays a "'compact(...)' variables existence". However the code is fully functional:
[3v4l link](https://3v4l.org/dW5jf)

Expected behaviour
--
The inspection should allow to input strings and even array of strings without being displayed. Maybe the inspection could search the content of the variable to check if the strings inside match with defined variables...

**Comment by kalessil** [comment]
Fixed!
